android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -52,6 +57,11 @@ class MusicService : HeadlessJsTaskService() {
     private var notificationCapabilities: List<Capability> = emptyList()
     private var compactCapabilities: List<Capability> = emptyList()
 
+    override fun onStartCommand(intent: Intent?, flags: Int, startId: Int): Int {
+        startTask(getTaskConfig(intent))
+        return START_STICKY
+    }
+
```

Comment:
We overwrite the `JSTaskService` here, and returning `START_STICKY`, which causes the service to try and recover itself if it dies for any reason.